ghib.js

```diff
@@ -0,0 +1,45 @@
+// adding an event listener to load content automatically when the page is loaded
+document.addEventListener("DOMContentLoaded", () => {
+    // console.log("page loaded:")
+
+    fetchFilms()
+    selectFilm()
+})
+
+//retrieving the data from the api to the js code
+const fetchFilms = () => {
+    let filmRequest = new XMLHttpRequest()
```

Comment:
This is the proper use of `XMLHttpRequest()` however you seemed to have forgotten the fact that the function on `.onreadystatechange` property of the `filmRequest` function won't be called until the state of the network request changes and the body of the if statement won't run until the request is `DONE`. This is asynchronous behavior. We don't know whether the request will success or fail nor do we know how long it is going to take. This is why we leave a function on `.onreadystatechange` so that it is called once that happens.